src/WebJobs.Extensions.DurableTask/LocalHttpListener.cs

```diff
@@ -80,11 +90,8 @@ public async Task StartAsync()
                         this.durableTaskOptions.HubName,
                         functionName: string.Empty,
                         instanceId: string.Empty,
-                        message: $"Failed to open local socket {availablePort}. This was attempt #{numAttempts} to open a local port.");
+                        message: $"Failed to open local port {availablePort}. This was attempt #{numAttempts} to open a local port.");
                     numAttempts++;
-                    var random = new Random();
```

Comment:
This randomized delay is no longer necessary as we don't have a semi-deterministic mechanism for selecting ports anymore, so it doesn't matter if multiple apps are port-searching at the same time.